internal/bminventory/inventory.go

```diff
@@ -121,7 +149,16 @@ const ignitionConfigFormat = `{
           "name": "root"
       },
       "contents": { "source": "data:,{{.AGENT_MOTD}}" }
-    }]
+	},
+	{
+	  "overwrite": true,
+	  "path": "/etc/pki/ca-trust/source/anchors/rh-it-root-ca.crt",
```

Comment:
There is a Systemd process on boot that will update the CA from the directory `/etc/pki/ca-trust/source/anchors` :
See https://github.com/coreos/fedora-coreos-config/pull/120